.changelog/1709.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+internal/config: add `${app.name}` variable
```

Comment:
This should just be `config:` 

We're working on documenting all of our preferences here. :)